config/importer/gajim.go

```diff
@@ -296,13 +296,13 @@ func mergeAccountInformation(ac gajimAccountInfo, s gajimOTRSettings, s2 map[str
 		}
 	}
 
+	res.Proxies = make([]string, 0)
 	if ac.proxy != "" {
-		res.Proxies = []string{ac.proxy}
-		res.RequireTor = true
```

Comment:
Well, the previous behavior was to set RequireTor in this circumstance because it meant the proxy was a tor proxy. We just assume any proxy set was a tor proxy. But in this case, we can just remove the RequireTor line I think.